### ISSUE TITLE
UI Elements Improvements

### ISSUE THREAD

**Issue by CryingWelkin** [lkarlslund/Adalanche#35]
There is a problem in the dialog boxes (modals) that display information; suppose you expand its height over the appbar of the browser:

![image](https://github.com/lkarlslund/Adalanche/assets/100922318/6104984a-8bcf-4255-aca2-0a9093c31ba3)

Afterwards its becomes impossible to control it and drag it anywhere, since that is only at the top of border of the modal/dialog box:

![image](https://github.com/lkarlslund/Adalanche/assets/100922318/1df02f00-f88f-4b07-8c29-5ce219d9286d)

One solution is to allow that drag element/functionality for all the borders.

**Comment by lkarlslund** [comment]
Yes, the windows have been a huge problem since introduction. I'm not a JavaScript wizard, so it's still not solved.

**Comment by PedantHTB** [comment]
@lkarlslund No worries just wanted to inform you about it. I will try and see if I can do that, but I am also no JS wizard. Only a Blazor one :P

**Comment by PedantHTB** [comment]
Fixed in https://github.com/lkarlslund/Adalanche/pull/37

**Comment by PedantHTB** [comment]
There are other cases where https://github.com/lkarlslund/Adalanche/pull/37 fails. Will reopen until those are fixed.
